tests/Microsoft.Identity.Web.Test.Common/TestHelpers/MsalTestTokenCacheProvider.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Caching.Memory;
+using Microsoft.Extensions.Options;
+using Microsoft.Identity.Web.TokenCacheProviders;
+using Microsoft.Identity.Web.TokenCacheProviders.InMemory;
+using System.Threading.Tasks;
+
+namespace Microsoft.Identity.Web.Test.Common.TestHelpers
+{
+    public class MsalTestTokenCacheProvider : MsalAbstractTokenCacheProvider
+    {
+        public IMemoryCache MemoryCache { get; }
```

Comment:
Should there be one IMemoryCache per cache key?
Meaning should MemoryCache be a Dictionary<string, IMemoryCache>() ?

Reply:
synced off line.

---
In reply to: [385982535](https://github.com/AzureAD/microsoft-identity-web/pull/39#discussion_r385982535) [](ancestors = 385982535)